sublime-package.json

```diff
@@ -16,17 +16,22 @@
                     "typescript": {
                       "additionalProperties": false,
                       "properties": {
-                        "serverPath": {
+                        "tsdk": {
                           "type": "string",
-                          "description": "Path to tsserverlibrary.js / tsserver.js / typescript.js."
+                          "description": "Path to tsserver's lib directory."
                         },
-                        "localizedPath": {
-                          "type": "string",
-                          "description": "Path to lib/xxx/diagnosticMessages.generated.json"
-                        }
                       }
-                    }
-                  }
+                    },
+                    "ignoreTriggerCharacters": {
+                      "type": "array",
+                      "items": {
+                        "type": "string",
+                        "maxLength": 1
```

Comment:
I think maxLength should be removed.

Here is how volar uses `ignoreTriggerCharacters`:
```
		if (options.ignoreTriggerCharacters) {
			server.completionProvider.triggerCharacters = server.completionProvider.triggerCharacters
				?.filter(c => !options.ignoreTriggerCharacters!.includes(c));
		}
``` 
It looks like there is no reason to limit to 1.

Reply:
Why do you think so? It's a length limit of each single string in the array. Trigger chars per spec can he single characters

Reply:
ah yes, my bad.
I tought it was the lenght of the array :+1: